justfile

```diff
@@ -65,7 +59,10 @@ mvn MVN=default_mvn TARGET=default_target PROFILES=default_profiles *FLAGS=defau
 
     {{MVN}} {{FLAGS}} install --projects liftwizard-utility/liftwizard-checkstyle
 
-    echo "Running: {{MVN}} {{FLAGS}} {{TARGET}} {{PROFILES}}"
+    bold=$(tput bold)
+    normal=$(tput sgr0)
+
+    echo "Running: ${bold}{{MVN}} {{FLAGS}} {{TARGET}} {{PROFILES}}${normal}"
```

Comment:
Just's own output is bold, so I'm experimenting with logging a single bold line from shebang recipes.